tests/unit/multiclass/BaggingMachine_unittest.cc

```diff
@@ -27,11 +30,169 @@
 using namespace shogun;
 using ::testing::Return;
 
+class BaggingMachine : public ::testing::Test
+{
+public:
+  CDenseFeatures<float64_t>* features_test;
+  CDenseFeatures<float64_t>* features_train;
+  CMulticlassLabels* labels_train;
+  SGVector<bool> ft;
+  virtual void SetUp()
+  {
+    load_data();
+  }
+ 
+  virtual void TearDown()
+  {
+    SG_UNREF(features_train);
+    SG_UNREF(features_test);
+    SG_UNREF(labels_train);
+  }
+
+	void load_data()
+	{
+    sg_rand->set_seed(1);
```

Comment:
formatting!
Use the style checker in travis to get the command that converts all your changes to the correct formatting, or do it manually